tests/admin_views/test_forms.py

```diff
@@ -27,7 +28,21 @@ def test_inactive_user(self):
         self.assertEqual(form.non_field_errors(), ["This account is inactive."])
 
 
-class AdminFormTests(SimpleTestCase):
+@override_settings(ROOT_URLCONF="admin_views.urls")
+class AdminFormTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.superuser = User.objects.create_superuser(
+            username="super", password="secret", email="super@example.com"
+        )
+
+        cls.active_user = User.objects.create_user(
+            username="activeuser", password="password", is_active=True
+        )
+
+    def setUp(self):
+        self.client.force_login(self.superuser)
```

Comment:
I think we have existing test classes which are already well setup for this e.g.
```diff
--- a/tests/admin_views/tests.py
+++ b/tests/admin_views/tests.py
@@ -1508,6 +1508,16 @@ class AdminViewBasicTest(AdminViewBasicTestCase):
         self.assertContains(response, "<h1>Change article</h1>")
         self.assertContains(response, "<h2>Article 2</h2>")
 
+    def test_change_view_error_subtitle_with_object(self):
+        response = self.client.post(
+            reverse("admin:admin_views_article_change", args=(self.a1.pk,)),
+            {}
+        )
+        self.assertContains(
+            response,
+            "<title>Error: Article 1 | Change article | Django site admin</title>",
+        )
+
     def test_view_subtitle_per_object(self):
```
I don't expect you need to adjust any `setUpTestData` or `setUp`

Reply:
Okay.. I have changes this accordingly